lib/bootloader_pvm.pm

```diff
@@ -162,13 +168,13 @@ sub prepare_pvm_installation {
         return (bootloader_pvm::prepare_pvm_installation $boot_attempt);
     }
 
-    assert_screen("run-yast-ssh", 300);
+    assert_screen("run-yast-ssh", 300) unless get_var('AGAMA_TEST');
```

Comment:
I think we can return before this line, for code after that are not needed for Agama test.

Reply:
817b033f2